examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
Both of the text and the images IMO, though not necessarily as an example (eg. it could be a FAQ entry instead).